src/style.css

```
@@ -0,0 +1,62 @@
+* {
+  margin: 0;
+}
+
+.star {
+  height: 16px;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  display: inline-block;
+}
+
+.star > .stars__star {
+  background-image: url(images/star.svg);
+}
+
+.stars--5 > .stars__star {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 > .stars__star:nth-of-type(-n+1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-of-type(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > .stars__star:nth-of-type(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > .stars__star:nth-of-type(-n+4) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.star > .stars__star {
  background-image: url(images/star.svg);
}

.stars--5 > .stars__star,
.stars--1 > .stars__star:nth-of-type(-n+1),
.stars--2 > .stars__star:nth-of-type(-n+2),
.stars--3 > .stars__star:nth-of-type(-n+3),
.stars--4 > .stars__star:nth-of-type(-n+4) {
  background-image: url(images/star-active.svg);
}
```
separate selectors with same styles by comma